data.sql

```diff
@@ -5,3 +5,69 @@ VALUES ('Agumon', '2020-02-03', 0, 'true', 10.23),
 ('Gabumon', '2018-11-2018', 2, 'false', 8)
 ('Pikachu', '2021-01-7', 1, 'false', 15.04)
 ('Devimon', '2017-05-12', 5, 'true', 11);
+
+-- Insert new data to the animale table
+INSERT INTO
+animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+	('Charmander', '2020-02-08', 0, 'false', 11),
+	('Plantmon', '2021-11-15', 2, 'true', 5.7),
+	('Squirtle', '1993-04-02', 3, 'false', 12.13),
+	('Angemon', '2005-07-12', 1, 'true', 45),
+	('Boarmon', '2005-07-07', 7, 'true', 20.4),
+	('Blossom', '1998-10-13', 3, 'true', 17),
+	('Ditto', '2022-05-14', 4, 'true', 22);
+
+-- First Transaction
+    -- Begin the transaction
+    BEGIN;
+    -- Set the condition (update the species column to be inspecified)
+    UPDATE animals
+    SET species = 'unspecified';
+    -- Then undo these changes by roll back
+    ROLLBACK;
+
+-- Second Transaction
+  -- Begin the transaction
+      BEGIN;
+  -- Set the condition (update the species column Where the name ends with 'mon')
+    UPDATE animals
+    SET species = 'digimon'
+    WHERE name LIKE '%mon';
+  -- Set the condition (update the species column Where the name doesn't end with 'mon')
+    UPDATE animals
+    SET species = 'pokemon'
+    WHERE name NOT LIKE '%mon';
```

Comment:
- [ ] Kindly here the requirements say `Update the animals table by setting the species column to pokemon for all animals that don't have species already set.` So you have to check if it's equal to NULL.